src/courses/views.py

```diff
@@ -581,6 +583,21 @@ def ajax_progress_chart(request, user_id=0):
                 {'x': day + 1, 'y': xp}
             )
 
+        # SAT + SUN are masked by default from numpy.busday_offset. Dont have to worry about it not being excluded
+        # have to use datetime.now() because get_date will sometimes be ahead by 1 day when using get_date() + fraction_complete()
+        today = make_aware(datetime.datetime.now())
```

Comment:
according to this github answer https://stackoverflow.com/a/10784482. timezone.now is ```It's based on utc instead of your default timezone.```

Reply:
will use timezone

Reply:
actually the date needs to be timezone aware to be used with ```user.profile.xp_to_date```. But ```timezone.now``` is only timezone aware if [USE_TZ=True](https://docs.djangoproject.com/en/5.0/ref/settings/#std-setting-USE_TZ). Very sure it will crash if its ```False```

Reply:
yep removed comment.

using ```timezone.localtime``` to match ```ExcludeDate.date``` and ```Semester.first_date + last_date``` as they all use ```DateField``` which gets converted to local timezone.